scenarios/prospect/manage-prospectings.json

```diff
@@ -0,0 +1,22 @@
+{
+  "scenarios": [
+    {
+      "label": "Prospects - Case Overview - Menu Displayed",
```

Comment:
Should be named `Manage Prospects - Case Overview - Case Action Menu Displayed` or `Manage Prospects - Case Overview - Case Action Menu`.

Reply:
Done: https://github.com/compucorp/cases-product-suite-backstopjs/pull/4/commits/37e77133f29f04c253daacfe0a0b0f6c29b54d62